test/token/ERC721/ERC721Token.test.js

```diff
@@ -11,30 +11,28 @@ require('chai')
   .use(require('chai-bignumber')(BigNumber))
   .should();
 
-contract('ERC721Token', function (accounts) {
+contract('ERC721Token', function ([owner, anyone, ...otherAccounts]) {
```

Comment:
Lodash is used here, so such definition doesn't work: `function ([_, owner, anyone, ...otherAccounts])`
But is it necessary to use "_" as an argument at all and what's the reason for including it in other tests?

Reply:
I don't think the reasoning behind `_` is explained anywhere yet, though it will be in the testing guide (#1077). The idea is to only used named accounts (`owner`, `anyone`, etc.), putting `_` first in the array means none of the named accounts will be `accounts[0]`, the default account, so any accidental errors where the `from` parameter is missing should end up in a test failure, and therefore be detected.